bin/deploy.sh

```diff
@@ -32,7 +32,7 @@ rm -rf seeds/test
 rm seeds/development.js
 rm seeds/test.js
 rm seeds/shared/delete-everything.js
-npm ci
+npm ci --only=production
```

Comment:
When we deploy the API, we should only push the production dependencies.  Our dev dependencies aren't useful and there's no good argument for putting unused code into production!

Reply:
nice, agreed

Reply:
also a good test that our dependencies are in the right place (dev vs. prod) in package.json